src/raster/rasterband.rs

```diff
@@ -449,6 +449,13 @@ impl<'a> RasterBand<'a> {
             Ok(RasterBand::from_c_rasterband(self.dataset, overview))
         }
     }
+
+    /// Return the unit of the rasterband.
+    /// If there is no unit, the empty string is returned.
```

Comment:
I can't find this in the GDAL docs, can we have an `assert_ne!(str_ptr, std::ptr::null())` or something like that?

Reply:
[f747c81](https://github.com/georust/gdal/pull/271/commits/f747c817b3dc39d0e3803519f9c58052ae961c4c)